### DISCUSSION TITLE
How to launch a coqui server in a remote server

### DISCUSSION THREAD

**Discussion by iRanadheer** [coqui-ai/TTS#1955] [General Q&A]
hey,

i was wondering if there's a way to launch coqui server from the remote machine and open in our browser? I've tried port forwarding, but didnt work.

**Comment by thorstenMueller** [comment]
Not sure if i understand your question right. You start Coqui TTS server process on a machine (your local network or remote machine) and try to reach http://<remote-ip>:5002? If this isn't working, there might be a network routing or firewall issue in your setup.

**Comment by Ca-ressemble-a-du-fake** [comment]
@iRanadheer Did you try the printed ipv6 address ?

Here (Ubuntu 22.04) I am accessing the "training machine" remotely via SSH withOUT port forwarding. I run coqui server like this (coqui tts version 0.8.0) : `python3 ~/coquiTTS/TTS/TTS/server/server.py`
Then it prints an IPV6 address that I use :

> INFO:werkzeug:WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 Running on all addresses (::)
  Running on http://[::1]:5002
 Running on http://[ip:v6:ad:dre:ss]:5002 <<<<<----- I am ctrl clicking on it and it opens on my browser
INFO:werkzeug:Press CTRL+C to quit
